dspace-api/src/test/data/dspaceFolder/config/local.cfg

```diff
@@ -108,3 +108,10 @@ plugin.sequence.java.util.Collection = \
     java.util.LinkedList, \
     java.util.Stack, \
     java.util.TreeSet
+
+
+# Enable IRUS tracker for IRUS tests
+stats.tracker.enabled = true
+
+dspace.hostname = localhost
+dspace.url =  http://localhost
```

Comment:
These two settings should be removed. They are no longer valid configurations in DSpace (as noted above)

Reply:
Removed these config params